### PR TITLE
feat: add ens name and avatar support

### PR DESCRIPTION
**Pull request by bigint**
# Add ens name and avatar support
  
## Changes 👷‍♀️

- Removed react-jazzicon which is unmaintained for 3 years
- Add davatar package which has ens avatar support
- Add useENS hook

image

image
  
## How to test 🧪

- Login with an account that has ens name
- Check the navbar
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

File: features/account/Account.tsx [hunk 1]
```diff
@@ -198,7 +202,7 @@ export function AccountModal({ close }: ModalProps) {
               </Flex>
               <Flex sx={{ alignItems: 'center' }}>
                 <Box mr={2}>
-                  <Jazzicon diameter={28} seed={jsNumberForAddress(account)} />
+                  <Davatar size={30} address={account} generatedAvatarType="jazzicon" />
```
Comment (bigint) [inline features/account/Account.tsx:205]:
```suggestion
                  <Davatar size={28} address={account} generatedAvatarType="jazzicon" />
```

File: helpers/useENS.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { ethers } from 'ethers'
+import { useEffect, useState } from 'react'
+
+export function useENS(address: string) {
+  const infuraUrlBackend = `https://mainnet.infura.io/v3/${process.env.INFURA_PROJECT_ID_BACKEND}`
```
Comment (adamskrodzki) [inline helpers/useENS.ts:5]: this should not be hardcoded here, 

search useAppContext and and context property for details

**Review by adamskrodzki — CHANGES_REQUESTED** [review]

**Comment by adamskrodzki** [comment]
@bigint  closing for now, but we will implement this after new year. Thanks for contribution

**Comment by brantlymillegan** [comment]
ser we're trying to get to 420 ENS integrations by the end of 2021

**Pull request closed without merging**